detection_rules/version_lock.py

```diff
@@ -146,6 +223,7 @@ def add_changes(r, *msg):
                         "rule_name": existing_rule_lock["rule_name"],
                         "sha256": existing_rule_lock["sha256"],
                         "version": existing_rule_lock["version"],
+                        "type": existing_rule_lock["type"]
```

Comment:
from 1139a7e948e04ea195e3337b537fd06ea2b0908

Validation was (rightfully) failing because the types were not added to the previous entries in the most recent lock. This should address that bug. Also, manually added the types to those missing entries within the lock